refresh/index.js

```diff
@@ -1041,6 +1043,18 @@ module.exports = Generator.extend({
         var swaggerFilename = this.destinationPath('definitions', `${this.projectName}.yaml`)
         this.conflicter.force = true
         this.fs.write(swaggerFilename, YAML.safeDump(this.swagger))
+
+        // Append items to .gitignore that may have been added through model gen process
+        var gitignoreFile = this.fs.read(this.destinationPath('.gitignore'))
+        for (var item in this.itemsToIgnore) {
+          // Ensure we only add unique values
+          if (gitignoreFile.indexOf(this.itemsToIgnore[item]) === -1) {
+            this.fs.append(
+              this.destinationPath('.gitignore'),
+              this.itemsToIgnore[item]
+            )
+          }
+        }
```

Comment:
@tunniclm Well I first noticed it in the integration tests that multiple references to an iOS SDK were being added to the `.gitignore`. It can also happen with multiple runs of the `yo swiftserver:model` command, which is possible right now, but maybe not encouraged, not sure?